src/components/display/SkeletonTableBody.tsx

```diff
@@ -15,7 +16,7 @@ export const SkeletonTableBody = (props: SkeletonTableBodyProps) => {
   // Build each of the rows we want
   const skeletonTableRows = React.useMemo(() => {
     return Array.from(new Array(rowCount), () => (
-      <SkeletonTableRow columnCount={columnCount} />
+      <SkeletonTableRow columnCount={columnCount} key={uuid()} />
```

Comment:
@jmhar3 instead of installing a package, I think this is an instance where passing an `index` to the key prop is OK. typically it's recommended not to do this as if the underlying array's sort/order changes our components end up with the wrong keys, but that isn't a concern here

the function passed as `Array.from`'s second argument is itself passed the current element's value, so we can do this...

```jsx
const skeletonTableRows = React.useMemo(() => {
  return Array.from(new Array(rowCount), (rowNumber) =>  ...)
}, [rowCount])
```